testing/Cargo.toml

```diff
@@ -6,7 +6,8 @@ workspace = ".."
 build = "build.rs"
 
 [dependencies]
-askama = { path = "../askama", version = "*" }
+serde_json = "1.0"
```

Comment:
I made it optional in `askama/Cargo.toml` since obviously not all users will want the serde/serde_json dependencies. But that didn't seem as relevant to the end-to-end 'testing' directory. So this change currently requires serde-json and enables the askama serde-json feature in order to run the end-to-end tests. Of course I could wire up the end-to-end tests to also conditionally compile the json tests. Your call. ;-)